os/drivers/input/Kconfig

```diff
@@ -0,0 +1,27 @@
+menu "Touch Device Support"
+
+menuconfig TOUCH
+	bool "Touch Device Support"
+	default n
+	---help---
+		Drivers for Touch Devices. These drivers support
+		interfaces as defined in include/tinyara/input/touchscreen.h
+
+		This selection is necessary to enable support for touch drivers in
+		drivers/input
+
+if TOUCH
+
+config TOUCH_IST415
```

Comment:
Let's add this in choice, because we will not support several touch ic at the same time.
If you think just config is right, then need `default` value.